src/java/boa/types/proto/NamespaceProtoTuple.java

```diff
@@ -49,6 +49,15 @@
 
 		names.put("comments", counter++);
 		members.add(new BoaProtoList(new CommentProtoTuple()));
+		
+		names.put("Statments", counter++);
```

Comment:
The names of these new attributes are the wrong case.  The name should match the name in the .proto file - which for these 3 new ones are all lowercase.